Orm/Xtensive.Orm/Core/LockableBase.cs

```diff
@@ -16,35 +16,29 @@ namespace Xtensive.Core
   [Serializable]
   public abstract class LockableBase: ILockable
   {
-    private bool isLocked;
 
     /// <inheritdoc/>
-    public bool IsLocked
+    public bool IsLocked { [DebuggerStepThrough] get; private set; }
+
+    /// <summary>
+    /// Ensures the object is not locked (see <see cref="ILockable.Lock()"/>) yet.
+    /// </summary>
+    /// <exception cref="InstanceIsLockedException">Specified instance is locked.</exception>
```

Comment:
We don't specify anything here, like we do with extenson

```suggestion
    /// <exception cref="InstanceIsLockedException">The instance is locked.</exception>
```